docs/user-guide/cli-updatingcli.md

```diff
@@ -29,7 +33,15 @@ If you have an `@lts-incremental` version of Zowe CLI (Zowe v1.0.x - v1.8.x), yo
 
 4. Install the most recent `@zowe-v1-lts` version CLI and optional plug-ins. For more information, see [Installing CLI](./cli-installcli.md).
 
-5. **(Optional)** Reestablish your user profiles. Move the profile configuration files that you saved in Step 1 into the `~/.zowe/profiles` or `%homepath%\.zowe\profiles` folder on your computer.
+5. **(Optional)** If you deleted your profiles in Step 1, recreate the profiles that you need manually.
+
+6. **(Optional)** If you copied your profiles to a local directory in Step 1, follow these steps:
+
+   **a.**  Move the profile configuration files that you saved in Step 1 back to the `~/.zowe/profiles` or `%homepath%\.zowe\profiles` folder on your computer.
+
+   **b.** Issue the `zowe scs update` command to update profiles that are secured with the Secure Credential Store Plug-in.
+
+   **c.** Issue the command `zowe profiles update zosmf <my-profile-name> --user <my-username> --password <my-password>` to update z/osmf profiles to use the current option names.
```

Comment:
I think the steps look great overall.
The only caveat is that people will have to know what profiles have the wrong property/option names, i.e. they will have to know which profiles they created when using lts-incremental or lts-stable.
Hence why it's better to just recreate the profiles based on previous information 😋